src/Illuminate/Validation/ValidationServiceProvider.php

```diff
@@ -53,6 +53,11 @@ protected function registerValidationFactory()
      */
     protected function registerPresenceVerifier()
     {
+        // since 'validation.presence' is optional, make the 'db' binding optional as well
+        if (! isset($this->app['db'])) {
```

Comment:
I have checked and `isset($this->app['db'])` does exactly that, so I went for isset the same way it is used elsewhere in the code.